actioncable/test/channel/stream_test.rb

```diff
@@ -26,7 +26,9 @@ def send_confirmation
       transmit_subscription_confirmation
     end
 
-    private def pick_coder(coder)
+    private
+
+    def pick_coder(coder)
```

Comment:
Rails' style indents the methods after `private`.

Reply:
Updated 👍